_posts/news/2016-01-25-announcing-scalajs-0.6.6.md

```diff
@@ -0,0 +1,173 @@
+That was too long, and we apologize.
+In the future, we hope never to let more than 2 months elapse between consecutive releases.
+
+Besides bug fixes, this release brings several major improvements:
+
+* `js.TupleN`, a JS equivalent of Scala tuples
+* Support for JUnit
+* Better support of constructors of `@ScalaJSDefined` classes:
+  they can now have overloads, default parameters and varargs
+* A completely redesigned internal API for the linker, whose main visible impact should be reduced memory usage and improved speed
+* `js.ConstructorTag[C]`, a `ClassTag` equivalent to `js.constructorOf[C]`
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/tutorial/).
+
+## Release notes
+
+As a minor release, 0.6.6 is (almost, see Breaking changes below) backward source and binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.6 without change.
+However, it is not forward compatible: libraries compiled with 0.6.6 cannot be used by projects using 0.6.{0-5}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Breaking changes
+
+### Running in Node.js/PhantomJS
+
+Until Scala.js 0.6.5, the following sbt setting would, as a side-effect, enable Node.js or PhantomJS:
+
+    scalaJSStage in Global := FastOptStage
+
+As of Scala.js 0.6.6, this setting is redundant, as it is the default, but `run` and `test` will still use Rhino by default!
+To disable Rhino and use Node.js/PhantomJS in 0.6.6, use the following setting instead:
+
+    scalaJSUseRhino in Global := false
+
+### Tools API
+
+The [Tools API]({{ site.production_url }}/api/scalajs-tools/0.6.6/#org.scalajs.core.tools.package),
+(the API to the Scala.js linker and optimizer) have been completely redesigned.
+Users of this API, such as Scala.jsFiddle, will experience breakages.
+Contact us if you need help porting your code.
+
+## Improvements
+
+### JavaScript Tuple types
+
+In the same spirit as `js.Function`s, `js.Array`, etc., we have added a series of types representing JavaScript "tuples": `js.Tuple2` to `js.Tuple22`.
+The JavaScript representation of a tuple is, in fact, a heterogeneous array of fixed length.
+
+For example, the value
+
+    val t = js.Tuple2(45, "hello")
+
+corresponds the JavaScript value
+
+    [45, "hello"]
+
+You should use `js.Tuple`s in facades to JavaScript libraries that take and return tuples as heterogeneous arrays.
+
+JavaScript tuples are convertible back and forth to Scala tuples, and you can use deconstruction to extract all values of a JavaScript at the same time.
+For example:
+
+    val scalaT: (Int, String) = t
+    val t2: js.Tuple2[Int, String] = scalaT
+    val js.Tuple2(a, b) = t2
+
+### Support for JUnit
+
+It is now possible to write JUnit tests for Scala.js!
+We have ported the entire JUnit API to Scala.js, so that it is now possible to write cross-compiling unit tests with JUnit.
```

Comment:
I would remove the word "entire". I think `We have ported the JUnit API to Scala.js, ...` should be good.